### DISCUSSION TITLE
quasar dev fails after building Capacitor app

### DISCUSSION THREAD

**Discussion by Dylan190774** [quasarframework/quasar#11099] [CLI - Capacitor / Cordova modes]
I'm building an app and during development I test and debug this with hot reloading in a browser with 
`quasar dev`

Then once in a while, when I'm satisfied with the results, I run the app on a real device, with:
```
quasar build --mode capacitor -T android
cd src-capacitor
npx cap open android
```

This opens Android Studio so I can run the app. This works ok, but after I do this the original browser-app reloads and says: 

`Module not found: Can't resolve imported dependency "@capacitor/core"  `

I find this behaviour quite annoying, since the only way to start debugging in the browser again is to restart the `quasar dev` again, which takes quite a while and opens a new browser tab, which I do not need.

Is there any way to use Quasar for real hybrid app? Maybe not mix the temporary builds in the same folder?

**Comment by hawkeye64** [comment]
Are you using yarn or npm and what version? Also, what Node version?

**Reply by Dylan190774** [reply]
Operating System - Windows_NT(10.0.19043) - win32/x64
NodeJs - 12.22.1

Global packages
  NPM - 6.14.12
  yarn - 1.22.4
  @quasar/cli - 1.2.2
  @quasar/icongenie - 2.2.0
  cordova - 9.0.0 (cordova-lib@9.0.1)

**Reply by hawkeye64** [reply]
Everything looks good.
When I did an Andriod app, I didn't use the browser per se as you mention above.
Once your Android is connected to your computer and running the app, you can have Chrome display what's on the mobile screen (HMR included)

**Reply by Dylan190774** [reply]
I find it much easier (and faster) to debug an app in a browser tab than on remote devtools, emulator or a device itself. For example, I use a MapboxGL map-object in my app, which doesn't show in the remote devtools.
With Cordova I never had this problem because this didn't need any special Cordova imports to function, so I could just leave the hot reload on and continue after building the app.
But I guess I just have to get used to this then...
